src/content/translations/pt-br/developers/docs/standards/tokens/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Padrões de token
-description: Token
+description:
```

Comment:
is `description` non-required?

Reply:
Missed adding this back in. Should be required. 
```suggestion
description: Token
```